profiles/targets/clang/packages

```diff
@@ -0,0 +1,9 @@
+# Copyright 1999-2016 Gentoo Foundation.
+# Distributed under the terms of the GNU General Public License, v2
+# $Id$
+
+*sys-devel/llvm
+*sys-devel/clang
+*sys-libs/libcxx
+*sys-libs/libcxxrt
+*sys-libs/llvm-libunwind
```

Comment:
I'm really not sure if this profile is ready to be used in building stages. The Linux kernel itself can't be built by clang as-is.